### PR TITLE
[CN-1134] Notes on restoring backup into an existing PersistentVolume

### PR DESCRIPTION
**Pull request by semihbkgr**

### CODE REVIEW

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 1]
```diff
@@ -173,6 +173,7 @@ include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 
 NOTE: For restore to be successful you have to give restore configuration at the `Hazelcast` custom resource creation. Otherwise, triggering restore on currently running members will likely to fail. 
 
+NOTE: It is not allowed to override the existing restore data with a backup. This means that if there is a PersistentValue left from the previous Hazelcast cluster, you cannot restore a backup into this PersistentValue.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/backup-restore.adoc:176]: Two notes one after another is not great; suggest making this a single note, as shown in the next comment

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 2]
```diff
@@ -215,7 +216,9 @@ include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 
 ====
 
-NOTE: For restore to be successful you have to give restore configuration at the `Hazelcast` custom resource creation. Otherwise, triggering restore on currently running members will likely to fail. 
+NOTE: For restore to be successful you have to give restore configuration at the `Hazelcast` custom resource creation. Otherwise, triggering restore on currently running members will likely to fail.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/backup-restore.adoc:219]:
```suggestion
[NOTE]
====
. You must configure restore at the creation of the `Hazelcast` custom resource. If this is not done, triggering restore on currently running members is likely to fail.
. You cannot override existing restore data using a backup. This means that, if a `PersistentValue` is left from the previousHazelcast cluster, you cannot use a backup to restore it.
====
```

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 3]
```diff
@@ -215,7 +216,9 @@ include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
...
+
+NOTE: It is not allowed to override the existing restore data with a backup. This means that if there is a PersistentValue left from the previous Hazelcast cluster, you cannot restore a backup into this PersistentValue.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/backup-restore.adoc:221]:
```suggestion

```

File: docs/modules/ROOT/pages/backup-restore.adoc (hunk 2 above)
Comment (rebekah-lawrence) [reply docs/modules/ROOT/pages/backup-restore.adoc:219]: I see this is used several times, but I'm not sure if I have correctly understood; just let me know if I haven't and we can work together to make sure the right message is there.

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/backup-restore.adoc (hunk 1 above)
Comment (semihbkgr) [reply docs/modules/ROOT/pages/backup-restore.adoc:176]: I was actually planning to remove first note 👍
I will update it after we decide.

**Comment by semihbkgr** [comment]
See the last decision in CN-1134
> make the backup data primary by overwriting the hot-restart.

> local backups are supported to be restored only once.

**Pull request closed without merging**